### PR TITLE
feat(Weverse): add presence 

### PR DESCRIPTION
**Pull request by Synxx12**
## Description 
Add presence for Weverse Kpop. The presence displays real-time updates about Kpop artists, including their current activities and recent posts on the platform.

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

  
Screenshot 1
Screenshot 2

</details>

### CODE REVIEW

File: websites/W/Weverse/metadata.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+		"id": "712745906876842025",
+		"name": "hanifhan1f"
+	},
+	"service": "Weverse",
+	"description": {
+		"en": "Displays the current live stream activity for a Weverse live page."
+	},
+	"url": "weverse.io",
+	"version": "1.0.0",
+	"logo": "https://res.cloudinary.com/debvitdiw/image/upload/v1726211085/Presences/weverse.jpg",
```
Comment (darkvillager2) [inline websites/W/Weverse/metadata.json:14]: Must be 512x512

File: websites/W/Weverse/presence.ts [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+}
+
+const presence = new Weverse({
+	clientId: "1284048129414402109",
+});
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		largeImageKey:
+			"https://res.cloudinary.com/debvitdiw/image/upload/v1726211085/Presences/weverse.jpg",
```
Comment (darkvillager2) [inline websites/W/Weverse/presence.ts:85]: Ditto

File: websites/W/Weverse/presence.ts [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+			},
+		];
+	} else {
+		presenceData.details = "Browsing Weverse";
+		presenceData.buttons = [
+			{
+				label: "Go to Weverse",
+				url: "https://weverse.io",
+			},
+		];
```
Comment (darkvillager2) [inline websites/W/Weverse/presence.ts:185]: This is not allowed.

**Review by darkvillager2 — CHANGES_REQUESTED** [review]

File: websites/W/Weverse/presence.ts [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+	getArtistName(): string {
+		return (
+			document
+				.querySelector(".LiveArtistProfileView_name_item__8W66y")
+				?.textContent?.trim() ?? "Unknown Artist"
+		);
+	}
+
+	getStreamTitle(): string {
+		const titleElement = document.querySelector("h2.TitleView_title__SSnHb");
```
Comment (theusaf) [inline websites/W/Weverse/presence.ts:15]: For better future support, consider using a selector without computer generated classes. For example,
```ts
"[class*=LiveArtistProfileView_name_item]"
```

**Review by theusaf — CHANGES_REQUESTED** [review]

File: websites/W/Weverse/presence.ts (hunk 4 above)
Comment (Synxx12) [reply websites/W/Weverse/presence.ts]: check my new commits

**Review by Synxx12 — COMMENTED** [review]
Done.

File: websites/W/Weverse/metadata.json [hunk 5]
```diff
@@ -0,0 +1,25 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.11",
+	"apiVersion": 1,
+	"author": {
+		"id": "712745906876842025",
+		"name": "hanifhan1f"
+	},
+	"service": "Weverse",
+	"description": {
+		"en": "Displays the current live stream activity for a Weverse live page."
```
Comment (Bas950) [inline websites/W/Weverse/metadata.json:10]:
```suggestion
		"en": "Enjoy every moment with artists on global fandom life platform Weverse."
```

**Review by Bas950 — CHANGES_REQUESTED** [review]
For the rest LGTM

**Review by Bas950 — APPROVED** [review]

**Review by theusaf — APPROVED** [review]

**Pull request merged**